commons-numbers-arrays/src/main/java/org/apache/commons/numbers/arrays/Norms.java

```diff
@@ -0,0 +1,451 @@
+        } else if (s2 != 0) {
+            // add s2, s3, c2, c3
+            final double s3Adj = s3 * SCALE_DOWN * SCALE_DOWN;
+            final double sum = s2 + s3Adj;
+            final double comp = ExtendedPrecision.twoSumLow(s2, s3Adj, sum) + c2 + (c3 * SCALE_DOWN * SCALE_DOWN);
+            return Math.sqrt(sum + comp);
+        }
+        // add s3, c3
+        return Math.sqrt(s3 + c3) * SCALE_DOWN;
+    }
+
+    /** Return a Euclidean norm value for special cases of non-finite input.
+     * @param v input vector
+     * @param start index to start examining the input vector from
+     * @return Euclidean norm special value
+     */
+    private static double euclideanNormSpecial(final double[] v, final int start) {
+        for (int i = start; i < v.length; ++i) {
+            if (Double.isNaN(v[i])) {
+                return Double.NaN;
+            }
+        }
+        return Double.POSITIVE_INFINITY;
+    }
+
+    /** Compute the maximum norm (also known as the infinity norm or L<sub>inf</sub> norm) of the arguments.
+     * The result is equal to \(\max{(|x|, |y|)}\), i.e., the maximum of the absolute values of the arguments.
+     *
+     * <p>Special cases:
+     * <ul>
+     *  <li>If either value is NaN, then the result is NaN.</li>
+     *  <li>If either value is infinite and the other value is not NaN, then the result is positive infinity.</li>
+     * </ul>
+     * @param x first input
+     * @param y second input
+     * @return maximum norm
+     * @see <a href="https://en.wikipedia.org/wiki/Norm_(mathematics)#Maximum_norm_(special_case_of:_infinity_norm,_uniform_norm,_or_supremum_norm)">Maximum norm</a>
+     */
+    public static double maximum(final double x, final double y) {
+        return Math.max(Math.abs(x), Math.abs(y));
+    }
+
+    /** Compute the maximum norm (also known as the infinity norm or L<sub>inf</sub> norm) of the arguments.
+     * The result is equal to \(\max{(|x|, |y|, |z|)}\), i.e., the maximum of the absolute values of the arguments.
+     *
+     * <p>Special cases:
+     * <ul>
+     *  <li>If any value is NaN, then the result is NaN.</li>
+     *  <li>If any value is infinite and no value is NaN, then the result is positive infinity.</li>
+     * </ul>
+     * @param x first input
+     * @param y second input
+     * @param z third input
+     * @return maximum norm
+     * @see <a href="https://en.wikipedia.org/wiki/Norm_(mathematics)#Maximum_norm_(special_case_of:_infinity_norm,_uniform_norm,_or_supremum_norm)">Maximum norm</a>
+     */
+    public static double maximum(final double x, final double y, final double z) {
+        return Math.max(
+                Math.abs(x),
+                Math.max(Math.abs(y), Math.abs(z)));
+    }
+
+    /** Compute the maximum norm (also known as the infinity norm or L<sub>inf</sub> norm) of the given values.
+     * The result is equal to \(\max{(|v_0|, ... |v_i|)}\), i.e., the maximum of the absolute values of the
```

Comment:
This should be `\(\max{(|v_0| + ... + |v_i|)}\)` to match the style of the other javadocs.

I rendered the javadocs and prefer the `\ldots`. Also use the length of the input n so the summation is to n-1.

Edit to `\(\max{(|v_0| +  \ldots +  |v_{n-1}|)}\)`.

```
rm -f target/maven-javadoc-plugin-stale-data.txt
mvn javadoc:javadoc
open target/site/apidocs/org/apache/commons/numbers/arrays/Norms.html
```

Reply:
I can't use '+' since that would produce the manhattan norm. I'll update the subscript and ellipsis, though.